lib/_stream_writable.js

```diff
@@ -57,8 +57,8 @@ function WritableState(options, stream, isDuplex) {
   // Duplex streams are both readable and writable, but share
   // the same options object.
   // However, some cases require setting options to different
-  // values for the readable and the writable sides of the duplex stream.
-  // These options can be provided separately as readableXXX and writableXXX.
+  // values for the readable and the writable sides of the duplex stream,
+  // e.g. options.readableObjectMode vs. options.writableObjectMode, etc.
```

Comment:
I think the changes will be unnecessary. I have no problem reading the this comment.

Reply:
Hi jc090218. Thanks for your feedback. 

I was able to make some sense of the 'XXX' comment eventually, after looking at the next few lines. I think I would have understood it more quickly were it as proposed.

You seem like someone that knows a lot about programming, way more than me. I'm a novice. Do you think others learning to program like me would benefit from the proposed change?

Reply:
Hi, jeswcollins,

I think you have a good point and perspective on the learner side of view. I would say when I learn programming that is basically what are the comments be worded. Thus everyone have their own way of learning, I do not think anyone can judge on each other's way of learning things. On my perspective, English is are not my native language and I recommend by wording thing like 'XXX' make thing simple and straightforward. 

Let us wait what the contributor/member thinks. :D